### DISCUSSION TITLE
Adding Basic Authentication to Hosted Allure Reports

### DISCUSSION THREAD

**Discussion by hatsu38** [allure-framework/allure2#2442] [Ideas & Feedback]
We are looking to implement a way to easily add Basic authentication to our Allure reports when hosting them outside of GitHub Pages. 

This is important for us because, although we want to use Allure Reports within our company, utilizing GitHub Pages for private access requires an Enterprise plan, which can be costly.

By having the ability to add Basic authentication, we can ensure a level of security for our reports without incurring additional expenses.

**Comment by noomorph** [comment]
@hatsu38, have you considered this solution: https://github.com/fescobar/allure-docker-service?
